fixtures/ext-types/lib/src/ext-types-lib.udl

```diff
@@ -1,12 +1,25 @@
 namespace imported_types_lib {
+    // Test the types individually
+    UniffiOneType get_uniffi_one_type(UniffiOneType val);
+    Animal get_another_animal(Animal val);
+    IpAddr get_another_ip(IpAddr addr);
+
+    // Test the CombinedType, which combines different external types together
     CombinedType get_combined_type(optional CombinedType? val);
 };
 
-// A type defined in a .udl file in the `uniffi-one` crate (ie, in
+// Some types defined in a .udl file in the `uniffi-one` crate (ie, in
 // `../../uniffi-one/src/uniffi-one.udl`)
 [External="uniffi-one"]
 typedef extern UniffiOneType;
 
+[External="uniffi-one"]
+typedef extern Animal;
+
+[External="uniffi-one"]
```

Comment:
Nit: this `External` type feels like it could be `ImportFrom` or `Use` which would fit the programming model. (FWIW understand why you chose External, but importing from a file is so common, External seems a bit strange).

Reply:
Yeah, I take full responsibility for `External` and happy to bikeshed it - it makes more sense from the view inside the scaffolding generator than from the view outside looking in.